packages/core/src/warnDuplicatePkg.ts

```diff
@@ -1,28 +1,82 @@
-interface Package {
+interface InstanceInfo {
   from: string;
   eager?: boolean;
   loaded?: number;
 }
 
-interface Packages {
-  [key: string]: {
-    [key: string]: Package;
-  };
+interface LoadedInstance {
+  [version: string]: InstanceInfo;
+}
+
+interface Package {
+  [pkgName: string]: LoadedInstance;
+}
+
+interface ToAlign {
+  [appName: string]: string;
 }
+declare let __webpack_share_scopes__: { [x: string]: any };
+
+const getWebpackScope = () => {
+  return __webpack_share_scopes__['default'];
+};
+
+const prepareUpdateCommands = (entries: [string, LoadedInstance][]) => {
+  const toAlign: ToAlign = {};
+  const visitedPackages: Record<string, string> = {};
+
+  entries.forEach(([pkgName, pkg]) => {
+    const loadedInstances = Object.entries(pkg);
+
+    if (loadedInstances.length > 1) {
+      loadedInstances.forEach(([, instanceInfo]) => {
+        const loadedApp = instanceInfo.from;
+
+        if (loadedApp && loadedApp !== 'insights-chrome' && !(visitedPackages[pkgName] === loadedApp)) {
+          if (toAlign[loadedApp]) {
+            toAlign[loadedApp] = `${toAlign[loadedApp]},${pkgName}`;
+          } else {
+            toAlign[loadedApp] = pkgName;
+          }
+
+          visitedPackages[pkgName] = loadedApp;
+        }
+      });
+    }
+  });
+
+  return toAlign;
+};
 
 /**
  * Warns applications using the shared scope if they have packages multiple times
  */
-export const warnDuplicatePkg = (packages: Packages) => {
+export const warnDuplicatePkg = () => {
+  const packages: Package = getWebpackScope();
   const entries = Object.entries(packages);
+  const updateCommands: ToAlign = prepareUpdateCommands(entries);
+
+  const pkgNames = Object.keys(updateCommands);
+  if (pkgNames.length) {
+    console.group(
+      `%c [SCALPRUM]: You have following packages that is being loaded into browser multiple times. You might want to align your version with the chrome one`,
+      'font-size:20px; background:#F9F9F9; color:#581845; padding:1px; border-radius:1px;'
+    );
 
-  entries.forEach(([pkgName, versions]) => {
-    const instances = Object.keys(versions);
-    if (instances.length > 1) {
+    const apps = Object.entries(updateCommands);
+
+    apps.forEach(([appName, packages]) => {
+      console.group(
+        `[SCALPRUM]: To align ${appName} application's package dependency versions to the chroming app instance version, please run following command`
+      );
       console.warn(
-        `[SCALPRUM]: You have ${pkgName} package that is being loaded into browser multiple times. You might want to align your version with the chrome one.`
+        `%c insights-interact run alignPackages --app=${appName} --packages=${packages} `,
```

Comment:
Can we this generic? Scalprum will be used in other places than just console.redhat.com. We don't want any product-specific references in this repository.

Reply:
Does it mean that I need to put this command log behind a feature flag or this command should not live in the scalprum at all? if the latter is the case, one thing that I can make is to return the toUpdate object and migrate this command warning to FEC instead.

Reply:
Any product-specific strings should not live in the codebase at all. We can make the message configurable and then set it within the chroming. But no HCC or RH-specific messages should not be anywhere in this codebase.